aries-site/src/pages/components/button.mdx

```diff
@@ -118,7 +118,7 @@ Icon buttons have an `a11yTitle`, `aria-label` or button `label` that describes
 **Best Practices:**
 
 - Limit to 2-3 descriptive words.
-- Use verbs first to describe the action such as “Hide/View”, "Switch/Unswitch", “Favorite/Unfavorite”, "Light/Dark" or “Like/Dislike".
+- Use verbs first to describe the action such as “Hide/View,” "Switch/Unswitch," “Favorite/Unfavorite,” "Light/Dark," or “Like/Dislike."
```

Comment:
This looks like a combination of regular double quotes and "smart" quote

Reply:
I fixed this now. using all regular quotes.